src/Engine/Objects/Player.cpp

```diff
@@ -2181,9 +2181,9 @@ int Player::GetMaxMana() const {
 
 //----- (0048E656) --------------------------------------------------------
 int Player::GetBaseAC() const {
-    int acc = GetActualAccuracy();
-    int accbonus = GetParameterBonus(acc);
-    int itembonus = GetItemsBonus(CHARACTER_ATTRIBUTE_AC_BONUS) + accbonus;
+    int spd = GetActualSpeed();
+    int spdbonus = GetParameterBonus(spd);
+    int itembonus = GetItemsBonus(CHARACTER_ATTRIBUTE_AC_BONUS) + spdbonus;
```

Comment:
Do we have an issue for this? We'll need to add a test.

Reply:
added #970